bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -52,6 +53,12 @@ pub enum Membership {
     Knocked,
 }
 
+#[derive(Debug, Clone, uniffi::Enum)]
+pub enum MediaEventsTimelineFilter {
+    ImageAndVideo,
+    FileAndAudio,
```

Comment:
A few remarks:

- can you add doc comments for these variants, please?
- it seems a bit arbitrary to join image with video, and file with audio. I suppose these come from ElementX requirements, but we're trying to make the SDK a bit more general (although arguably, the FFI layer is quite adhoc to EX apps). Would it make sense to split those into four variants: `Image`, `Video`, `File`, `Audio`, and accept an array of them in `media_events_timeline` instead? That way, embedders can compose these as they want to, and not have to add new variants in there.

Reply:
> can you add doc comments

sure thing 👍 

> Would it make sense to split those into four variants

Yep yep, makes sense to me, I'll make the change